packages/auth-client/test-setup.ts

```diff
@@ -18,11 +18,7 @@ Object.defineProperty(globalThis, 'crypto', {
   value: crypto,
 });
 
-Object.defineProperty(console, 'warn', {
-  value: jest.fn(),
+Object.defineProperty(global, 'console', {
+  writable: true,
+  value: { ...global.console, log: jest.fn(), warn: jest.fn(), error: jest.fn() },
```

Comment:
does this mean that no logs will be printed during test execution?

Reply:
Yes, it's to clean up the console by default when running the tests. However, the logs aren't lost and you can access them as needed